ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -860,13 +860,15 @@ tests:
     cluster_profile: aws
     test:
     - ref: network-flow-matrix-tests
+    - ref: openshift-e2e-test
```

Comment:
Running two test steps doesn't work well, if the first one fails the second one will never run and I don't think there's any way to change that behavior.

Reply:
There is a dependency here, the first one configures a firewall that is then validate by running openshift-tests, if the first step failed there is no need to run openshift-tests